android/src/main/java/com/alternadom/wifiiot/WifiIotPlugin.java

```diff
@@ -104,77 +102,55 @@ private void cleanup() {
         moWiFi = null;
         moWiFiAPManager = null;
     }
-
```

Comment:
Why is V1 embedding being removed? It should remain for backwards compatibility.